p4c_bm/shell.py

```diff
@@ -51,6 +51,10 @@ def get_parser():
     parser.add_argument('--meta-config', dest='meta_config',
                         default = "",
                         help="Path to json file specifying metadata config")
+#    parser.add_argument('--plugin', dest='plugins',
+    parser.add_argument('--plugin', dest='plugins', type=lambda s: [str(n) for n in s.split()],
```

Comment:
Can you use nargs='*' instead of this lambda function type (if all you want is a list) ?